src/test/resources/county-court-judgment/invalid-structure.json

```diff
@@ -0,0 +1 @@
+{ "this is invalid Json" }
```

Comment:
this isn't specific to CCJ, I recommend removing

Reply:
ok